### PR TITLE
Kg applications intro

### PR DESCRIPTION
**Pull request by danich1**
This section sets the scene for network representation learning and network applications.

### CODE REVIEW

File: content/04.network_applications.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ## Applying Knowledge Graphs to Biomedical Challenges
 
-1. Mention that these graphs can be used for discovery
-2. Mention representation learning (aka representing a graph as dense vectors for nodes and/or edges)
-3. 
+Knowledge graphs can be used to aid researchers in executing important biomedical tasks such as finding new treatments for FDA approved drugs [@doi:10.7554/eLife.26726] and predicting protein function in human tissues [@doi:10.1093/bioinformatics/btx252].
```
Comment (cgreene) [inline content/04.network_applications.md:3]:
```suggestion
Knowledge graphs can help researchers tackle many important biomedical tasks such as finding new treatments for FDA approved drugs [@doi:10.7554/eLife.26726] and predicting protein function in human tissues [@doi:10.1093/bioinformatics/btx252].
```

File: content/04.network_applications.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+Typically, the first step of these tasks involve modeling a knowledge graph's structural information (e.g., type of edges between nodes and how connected nodes are to each other).
```
Comment (cgreene) [inline content/04.network_applications.md:4]: Interestingly rephetio doesn't do this.

**Review by cgreene — COMMENTED** [review]
This feels very similar to me to the heading that comes with `### Unifying Techniques` below. I'd propose that you leave this one until the end, and then go back and revise it once the other sections below it get merged. I think then it may become clearer what this should be.

**Comment by danich1** [comment]
Ack did the merge wrong again. Closing this issue and opening a new one that will be easier to review.

**Pull request closed without merging**